BenchmarkVCFs/SimpleBenchmark.wdl

```diff
@@ -0,0 +1,596 @@
+    File? interval
+}
+
+# Main workflow: performs evaluation of call_vcf against base_vcf using vcfeval comparison engine,
+# over different analysis modes.
+workflow SimpleBenchmark {
+    input {
+        # VCF information
+        File base_vcf
+        File base_vcf_index
+        String base_output_sample_name
+        String? base_vcf_sample_name
+
+        File call_vcf
+        File call_vcf_index
+        String call_output_sample_name
+        String? call_vcf_sample_name
+
+        # Reference information
+        File ref_fasta
+        File ref_index
+        File ref_dict
+
+        # Subsetting inputs using intervals
+        Array[File] strat_intervals = []
+        Array[String] strat_labels = []
+        Int interval_padding = 0
+        String subset_gatk_tag = "4.2.3.0"
+
+        # Subsetting inputs using variant properties; Note SNP & INDEL are separated automatically later
+        Array[String] bcf_selectors = []
+        Array[String] bcf_labels = []
+
+        # Evaluation inputs
+        File? evaluation_bed
+        Array[String] score_fields = ["GQ"]
+
+        # Columns to add to output files
+        String? experiment_label
+        String? extra_column
+        String? extra_column_label
+
+        # Null File type -- do NOT assign; Needed until WDL has better support for null File values
+        File? NULL_FILE
+    }
+
+    # Create reference object
+    Reference reference = {"fasta": ref_fasta, "index": ref_index, "dict": ref_dict}
+
+    # Add in default "Whole Genome" empty interval list
+    Array[File] full_strat_intervals = flatten([[""], strat_intervals])
+    Array[String] full_strat_labels = flatten([[""], strat_labels])
+
+    # Make Stratifier objects
+    scatter (strat in zip(full_strat_labels, full_strat_intervals)) {
+        Stratifier stratifier_list = {"label": strat.left, "interval": if strat.right != "" then strat.right else NULL_FILE}
+    }
+
+    # Make VariantSelector objects, with default None, Het, and HomVar types
+    Array[String] default_bcf_selectors =  ["", "GT='het'", "GT='hom'"]
+    Array[String] default_bcf_labels = ["", "Het", "HomVar"]
+    Array[String] full_bcf_selectors = flatten([default_bcf_selectors, bcf_selectors])
+    Array[String] full_bcf_labels = flatten([default_bcf_labels, bcf_labels])
+    scatter (selection in zip(full_bcf_labels, full_bcf_selectors)) {
+        VariantSelector selector_list = {"bcf_label": selection.left, "bcf_selector": selection.right}
+    }
+
+    # Perform analysis over all Stratifiers
+    scatter (stratifier in stratifier_list) {
+        call SubsetVCF as SubsetEval {
+            input:
+                input_vcf=call_vcf,
+                input_vcf_index=call_vcf_index,
+                input_sample_name=call_vcf_sample_name,
+                reference=reference,
+                stratifier=stratifier,
+                interval_padding=interval_padding,
+                gatk_tag=subset_gatk_tag,
+                preemptible=3
```

Comment:
maybe make these default to 3, but still have a workflow-wide optional parameter to change to something else.

Reply:
Changed this to a global input